test/token/ERC20/StandardBurnableToken.test.js

```diff
@@ -0,0 +1,63 @@
+import assertRevert from '../../helpers/assertRevert';
+import shouldBehaveLikeBurnableToken from './BurnableToken.behaviour';
+const StandardBurnableTokenMock = artifacts.require('StandardBurnableTokenMock');
+
+contract('StandardBurnableToken', function ([owner, burner]) {
+  beforeEach(async function () {
+    this.token = await StandardBurnableTokenMock.new(owner, 1000);
+  });
+
+  shouldBehaveLikeBurnableToken([owner]);
+
+  describe('burnFrom', function () {
+    describe('on success', function () {
+      const amount = 100;
+
+      beforeEach(async function () {
+        await this.token.approve(burner, 300, { from: owner });
+        const { logs } = await this.token.burnFrom(owner, amount, { from: burner });
+        this.logs = logs;
+      });
+
+      it('burns the requested amount', async function () {
+        const balance = await this.token.balanceOf(owner);
+        assert.equal(balance, 900);
```

Comment:
We have to use bignumber comparisons. There's two options: 1) using `assert(balance.eq(900))`, or 2) using the `chai-bignumber` extension and writing assertions in either `expect` or `should` style.

2) is preferred but I'll accept 1).